sjb/config/test_cases/test_branch_origin_extended_conformance_install_system_containers.yml

```diff
@@ -95,12 +104,18 @@ extensions:
       script: |-
         docker_image_tag="$( cat ./ORIGIN_TAG )"
         docker_repository="${OS_IMAGE_PREFIX:-"openshift/origin"}"
-        docker inspect "${docker_repository}:${docker_image_tag}"
+        [[ "$(sudo atomic containers list -n --no-trunc -f runtime=runc -f image=${docker_repository}:${docker_image_tag} --json | jq '.[0].running')" == "true" ]]
```

Comment:
I would like that too :( Something for atomic guys. @ashcrow @giuseppe can you point me to a github repository where I can open upstream issue to extend the atomic command with json output?